src/compile-middleware.ts

```diff
@@ -116,11 +116,21 @@ export function babelCompile(
 
       let transformed;
       const contentType = getContentType(response);
-      const isComponentRequest = request.baseUrl === `/${componentUrl}` &&
-          !request.path.startsWith(`/${packageName}`);
+      let requestPath = request.path;
+      const isRootPathRequest = request.path.startsWith(`/${packageName}`);
```

Comment:
i guess we should make this startWith safer too?

Reply:
existing code? This logic shouldn't change on windows, so I'm going to defer for now.

Reply:
just add a todo?

Reply:
done